### PR TITLE
SJRK-197: Adding resources accordion

### PR DESCRIPTION
**Pull request by avtar**

### CODE REVIEW

**Comment by avtar** [comment]
These changes can be previewed at this site: https://affectionate-archimedes-d934d2.netlify.com/

**Comment by avtar** [comment]
P.S. if you click the [Pages changed Details link](https://github.com/fluid-project/sojustrepairit.org/pull/47/checks?check_run_id=57437571) you can preview the changes there as well:
https://deploy-preview-47--social-justice-repair-kit.netlify.com/resources

**Comment by michelled** [comment]
I think it would be good to add some hover and focus styling on the buttons and headings.

File: static/js/resources.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+    /** Toggle each category to hide or show their respective content(s).
+     * @param (evt) A jQuery event, happens each time the object is clicked
+     */
+    $('.resources-toggleItem').click(function (evt) {
+        var target = $(evt.delegateTarget);
+        var state = target.attr('aria-expanded') === 'true' ? true : false;
+        toggleCategory(target, !state);
+        // Remove focus indicator after clicks so that it doesn't linger indefinitely
+        this.blur();
```
Comment (avtar) [inline static/js/resources.js]: @michelled Is this a bad idea? Without this if you expand a resource then collapse it the focus indicator remains visible.

**Pull request merged**